lib/jekyll/commands/serve/servlet.rb

```diff
@@ -27,7 +27,7 @@ def search_file(req, res, basename)
 
         #
 
-        def do_GET(req, res)
+        def do_get(req, res)
```

Comment:
It is, this is an upstream method that we override upon subclassing.  So in order for us to fix this upstream would have to fix it.